ethereum/contracts/ShutdownSwitch.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+import "./interfaces/IWormhole.sol";
+
+contract ShutdownStructs {
+    struct VoterEntry {
+        address key;
+        bool enabled;
+    }
+}
+
+abstract contract ShutdownSwitch {
+    function requiredVotesToDisable() public view returns (uint16) {
+        return requiredNoVotes;
+    }
+
+    function numVotesToDisable() public view returns (uint16) {
+        return votesToDisable;
+    }
+
+    function enabledFlag() public view returns (bool){
+        return enabled;
+    }
+
+    event ShutdownVoteCast(address indexed voter, bool votedToEnable, uint16 numVotesToDisable, bool enableFlag);
+    event ShutdownStatusChanged(bool enabledFlag, uint16 numVotesToDisable);
+
+    function getWH() public virtual view returns (IWormhole);
+    function getChainId() public virtual view returns (uint16);
+
+    uint16 constant REQUIRED_NO_VOTES = 1;
+
+    ShutdownStructs.VoterEntry[20] voters;
+    uint32 guardianSetIndex = 0;
```

Comment:
We shouldn't need to hard-code this, we should be able to obtain this from a pre-existing getter method for https://github.com/certusone/wormhole/blob/dev.v2/ethereum/contracts/Getters.sol#L13-L15

Reply:
Well, now we are back to using a hard coded value of three. . .